### DISCUSSION TITLE
List all Working files in sidebar without scrolling

### DISCUSSION THREAD

**Discussion by ghost** [brackets-cont/brackets#167] [General]
I often have 40 odd working files. I would like to be able to see all my Working files, in the left hand sidebar, without having to scroll. At the moment only 9 working files are visible, in the Working Files space in the sidebar, and I have to scroll to see them all. Because the Brackets Folder space stops more than 9 files being displayed.

How can I get Brackets to show / list all my Working files.

![Screenshot 2022-03-19 at 2 45 50 pm](https://user-images.githubusercontent.com/81361036/159125779-35c98179-fa45-4921-940d-06c4e6e8fb4d.png)

**Comment by ghost** [comment]
I've just discovered the Taller Working Files extension. Sorted.

**Reply by Pulkit-Pandey-09** [reply]
Perfect. Best of  luck 👍
